src/seedsigner/models/settings_definition.py

```diff
@@ -51,6 +51,13 @@ class SettingsConstants:
         (COORDINATOR__KEEPER, "Keeper"),
     ]
 
+    BIP85__ENABLED_WITH_LOADING = "L"
+    ALL_BIP85_OPTIONS = [
+        (BIP85__ENABLED_WITH_LOADING, _mft("Enabled (With Loading)")),
+        (OPTION__ENABLED, _mft("Enabled (Generation Only)")),
```

Comment:
Maybe:
* "Generate, view-only"
* "Generate and use"

Naming things is always so hard.

I would prefer the more conservative option to be listed first.